django_probes/management/commands/wait_for_database.py

```diff
@@ -88,13 +90,20 @@ def add_arguments(self, parser):
                             help='which database of `settings.DATABASES` '
                                  'to wait for. Defaults to the "default" '
                                  'database.')
+        parser.add_argument("--command", "-c", default=None,
+                            action="store", dest='command',
+                            help='execute this command when database is up')
 
     def handle(self, *args, **options):
         """
         Wait for a database connection to come up. Exit with error
         status when a timeout threshold is surpassed.
         """
+        command = options.pop("command", None)
         try:
             wait_for_database(**options)
         except TimeoutError as err:
             raise CommandError(err) from err
+        if command:
+            command_list = shlex.split(command)
+            call_command(command_list[0], *command_list[1:])
```

Comment:
This is a bit cryptic for one-time readers. Maybe better like this?

```suggestion
            parts = shlex.split(command)
            executable, params = parts[0], parts[1:]
            call_command(executable, *params)
```